### PR TITLE
Add Moon filter

### PR DESCRIPTION
**Pull request by jagtalon**
Tried to get as close to the Moon as possible. 🚀

Hopefully this looks good! Here are the comparisons below:

## Real thing:
img_4501
img_4503

## CSS version:
screen shot 2015-11-14 at 20 45 55
screen shot 2015-11-14 at 20 46 46

### CODE REVIEW

**Comment by jagtalon** [comment]
@una added tests for this. Ready for review!

**Comment by jagtalon** [comment]
Just saw the review of lark. Will convert to RGB. :)

**Comment by jagtalon** [comment]
@una actually ready now!

File: source/scss/moon.scss [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ */
+@import 'shared';
+
+%moon,
+.moon {
+  @extend %filter-base;
+  filter: grayscale(1) contrast(1.1) brightness(1.1);
+
+  &:before {
+    background: rgb(180, 178, 178);
```
Comment (una) [inline source/scss/moon.scss]: Try: `rgb(160, 160, 160)` to make the shadows a bit darker. Otherwise, this PR looks :sparkles:*awesome*:sparkles:

Comment (jagtalon) [inline source/scss/moon.scss]: @una gotcha!

**Comment by jagtalon** [comment]
@una ready!

**Comment by una** [comment]
Sweet! This looks good.

Question: Have you ever tried rebasing commits? Rebase lets you merge your commits into a single neat one with a title like "Added Moon filter". You'll probably want to merge the last 6 like this: `git rebase -i HEAD~6` and keep the topmost one, then squash the rest (you can just use `s`) and terminal will walk you through the steps.

Here's a good reference: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by jagtalon** [comment]
@una Should be ready!

**Comment by una** [comment]
thanks!

**Pull request merged**